docs/src/content/docs/tutorials/split-payments.mdx

```diff
@@ -0,0 +1,296 @@
+
+Fetch the wallet addresses of the buyer, merchant, and your marketplace platform. In this example, we assume each party has their Open Payments-enabled account with a different account servicing entity.
+
+```ts
+const customerWalletAddress = await client.walletAddress.get({
+  url: 'https://gatehub.net/buyer'
+})
+
+const merchantWalletAddress = await client.walletAddress.get({
+  url: 'https://chimoney.io/merchant'
+})
+
+const platformWalletAddress = await client.walletAddress.get({
+  url: 'https://interledger.app/platform'
+})
+```
+
+### 2. Request Incoming Payment grants
+
+Request `incomingPayment` grants from your wallet authorization server as well as from the merchant wallet authorization server.
+
+<Tabs>
+<TabItem label = 'Merchant'>
+```ts 
+const merchantGrant = await client.grant.request(
+  {
+    url: merchantWalletAddress.authServer,
+  },
+  {
+    access_token: {
+      access: [
+        {
+          type: "incoming-payment",
+          actions: ["read", "create"],
+        },
+      ],
+    },
+  },
+);
+```
+</TabItem>
+<TabItem label = 'Platform'>
+```ts
+const platformGrant = await client.grant.request(
+  {
+    url: platformWalletAddress.authServer,
+  },
+  {
+    access_token: {
+      access: [
+        {
+          type: "incoming-payment",
+          actions: ["read", "create"],
+        },
+      ],
+    },
+  },
+);
+```
+</TabItem>
+</Tabs>
+
+### 3. Create Incoming Payment resources
+
+Create `incomingPayment` resources using the access tokens returned from the authorizations servers in step 2. Since this example assumes the merchant and the platform operator are using different account servicing entities, the `incomingPayment` resources will be created on the respective resource servers.
+
+In this example you will split the payment and give the merchant 99% on a $100 payment, while retaining 1% for your fee as the platform operator.
+
+<Tabs>
+<TabItem label = 'Merchant'>
+
+```ts
+const merchantIncomingPayment = await client.incomingPayment.create(
+  {
+    url: new URL(WALLET_ADDRESS).origin,
+    accessToken: INCOMING_PAYMENT_ACCESS_TOKEN
+  },
+  {
+    merchantWalletAddress: WALLET_ADDRESS,
+    incomingAmount: {
+      value: '9900',
+      assetCode: 'USD',
+      assetScale: 2
+    },
+    expiresAt: new Date(Date.now() + 60_000 * 10).toISOString()
+  }
+)
+```
+
+</TabItem>
+
+<TabItem label = 'Platform'>
+
+```ts
+const platformIncomingPayment = await client.incomingPayment.create(
+  {
+    url: new URL(WALLET_ADDRESS).origin,
+    accessToken: INCOMING_PAYMENT_ACCESS_TOKEN
+  },
+  {
+    platformWalletAddress: WALLET_ADDRESS,
+    incomingAmount: {
+      value: '100',
+      assetCode: 'USD',
+      assetScale: 2
+    },
+    expiresAt: new Date(Date.now() + 60_000 * 10).toISOString()
+  }
+)
+```
+
+</TabItem>
+</Tabs>
+
+### 4. Request quote grants
+
+Request two quote grants from the customer wallet authorization server, as there are two recipients on this payment.
```

Comment:
We just need a single quote grant on the sending wallet address side (the client can use the same quote grant to create multiple quotes)